molgenis-navigator/src/main/frontend/src/store/actions.js

```diff
@@ -59,34 +59,21 @@ function toEntity (item: any) {
 }
 
 /**
- * Get all ( first 1000 ) packages from the server
- * @returns {Promise}, on success resolves to list of 'all' packages, on failure resolves to error message
+ * Get a MOLGENIS rest api encoded query for the Package EntityType
+ *
+ * @param query
  */
-function getAllPackages () {
-  return new Promise((resolve, reject) => {
-    const uri = '/api/v2/sys_md_Package?sort=label&num=1000'
-    api.get(uri).then((response) => {
-      resolve(response.items)
-    }).catch((error) => {
-      reject(error)
-    })
-  })
+function getPackageQuery (query: string) {
+  return '/api/v2/sys_md_Package?sort=label&num=1000&q=id=q="' + encodeURIComponent(query) + '",description=q="' + encodeURIComponent(query) + '",label=q="' + encodeURIComponent(query) + '"'
 }
 
 /**
- * Query's the server of ( first 1000 ) packages.
```

Comment:
Same as above; the fact we only get the first 1000 items seems important to me, i suggest to keep this as part of the js doc, as this is not derivable from the function name or params